data.rs

```diff
@@ -203,8 +225,8 @@ impl PSP22Data {
         }])
     }
 
-    #[cfg(feature = "mintable")]
-    pub fn mint(&mut self, account: AccountId, value: u128) -> Result<Vec<PSP22Event>, PSP22Error> {
+    // Mint a `value` of new tokens to `to` account.
```

Comment:
In my experience, comments were told in 3rd person - sth like _"Mints a `value`"_, _"Returns ..."_, etc.

Reply:
Fixed